runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -4496,10 +4496,23 @@ TR::CompilationInfoPerThreadRemote::cacheResolvedMirrorMethodsPersistIPInfo(TR_R
    _resolvedMirrorMethodsPersistIPInfo->push_back(resolvedMethod);
    }
 
+void
```

Comment:
Suggestion: since the values are always going to be NULL you could change the name of this function to cacheNullClassOfStatic and don't pass the j9class as a parameter since it's known to be NULL.
I theory we should have used an unordered_set instead of an unordered_map, because the value is not needed, but that would prevent you from using the new methods you wrote for per-compilation-cache.

Reply:
Done.